README.md

```diff
@@ -72,6 +72,49 @@ That's it. Easy as apple pie.
     </script>
   </body>
 </html>
+```
+## React example (just loading videomail)
+```jsx harmony
+import React, { Component } from 'react';
+import { Helmet } from 'react-helmet';
+
+export default class VideomailReactExample extends Component{
+  state = {
+      videomailClient:null,
+  };
+
+  componentDidMount() {
+    (async() => {
+      console.log("waiting for videomail client library to load via CDN");
+      while(window.VideomailClient === undefined)
```

Comment:
what if the remote resource is not available due to some network issues? this would loop forever ...